src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManager.java

```diff
@@ -3,10 +3,13 @@
 import edu.kis.powp.jobs2d.command.DriverCommand;
 import edu.kis.powp.observer.Publisher;
 
+import java.util.List;
+
 public interface CommandManager {
     void runCommand();
     void clearCurrentCommand();
     void setCurrentCommand(DriverCommand driverCommand);
+    void setCurrentCommand(List<DriverCommand> driverCommand, String name);
```

Comment:
Is it now really necessary? You can use setCurrentCommand(DriverCommand driverCommand) now.